### PR TITLE
Added Source Sans Pro as a default font to design tokens.

### PR DESCRIPTION
**Pull request by antikus**
I guess Source Sans Pro should be a default font, shouldn't it?

### CODE REVIEW

**Comment by nickwynja** [comment]
When we first started rolling out Source Sans, we kept Helvetica the default and let people using the Enterprise jQuery controls [opt-in using `.personalize`](https://design.infor.com/code/ids-enterprise/latest/typography).

I think it probably makes sense to update the default here since there's no backwards-compatibility issues if a team just simply doesn't include the necessary font includes.  

@jmacaluso711 Foresee any issues with this?

**Comment by antikus** [comment]
I've also updated font-size property to have "px".

File: design-tokens/core/core.alias.yml [hunk 1]
```diff
@@ -34,11 +34,11 @@
     BORDER_COLOR: "{!PALETTE_GRAPHITE_4}"
 
     # Fonts
-    FONT_FAMILY_BASE: "helvetica, arial, san-serif"
+    FONT_FAMILY_BASE: "'Source Sans Pro', helvetica, arial, san-serif"
     FONT_FAMILY_MONOSPACE: "monospace"
     FONT_WEIGHT_NORMAL: 400
     FONT_WEIGHT_BOLD: 700
-    FONT_SIZE_BASE: "14"
+    FONT_SIZE_BASE: "14px"
```
Comment (nickwynja) [inline design-tokens/core/core.alias.yml]: There are instances where we use the `FONT_SIZE_BASE` value and do some sort of `calc()` based on the value or in many places add `px` in a `@mixin` in the current Enterprise components. So leaving this without `px` gives more overall flexibility.

**Comment by nickwynja** [comment]
This is good to go now. @clepore, who has been handling releases for the design tokens, is out this week. We can merge and release next week when he's back.

**Comment by nickwynja** [comment]
@antikus This is released in `1.3.1` now which is available on npm.

**Comment by antikus** [comment]
Thanks @nickwynja

**Pull request merged**